src/index.html

```diff
@@ -1,13 +1,275 @@
+      rel="stylesheet"
+    >
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="./images/favicon.svg"
+    >
+    <link rel="stylesheet" href="./styles/style.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <header class="header">
+      <div class="top-bar">
+        <a href="index.html" class="top-bar__logo-link">
+          <img
+            src="images/header_logo-2x.png"
+            alt="MyBike logo"
+            class="top-bar__logo"
+          >
+        </a>
+
+        <div class="top-bar__icons">
+          <a href="tel: +1 234 5555-55-55" class="icon icon--phone"></a>
+          <a href="#menu" class="icon icon--menu"></a>
+        </div>
+      </div>
+
+      <div class="header__bottom">
+        <h1 class="header__title">Take the Streets</h1>
+      </div>
+
+    </header>
+    <aside class="page__menu menu" id="menu">
+      <div class="top-bar menu__top">
+        <a href="index.html" class="top-bar__logo-limk">
+          <img
+            src="images/menu_logo_2x.png"
+            alt="MyBike logo"
+            class="top-bar__logo"
+          >
+        </a>
+
+        <a href="#" class="icon icon--close"></a>
+      </div>
+
+      <div class="menu__content">
+        <nav class="nav menu__nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link" href="#">Home</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#about-us">About Us</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#compare-bikes">Compare Bikes</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#details">Details</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#contacts">Contacts</a>
+            </li>
+          </ul>
+        </nav>
+
+        <a href="tel: +1 234 5555-55-55" class="menu__phone-number">+1 234 5555-55-55</a>
+
+        <a href="tel: +1 234 5555-55-55" class="menu__book-to-test">Book a test ride</a>
+      </div>
+    </aside>
+    <main class="main">
+      <section class="about" id="about-us">
+        <h2 class="about__title section-title">Move Free</h2>
+        <p class="about__content">Shift your ride, not gears. Find the fastest way to move in the city as
+          the bike adapts intuitively to power the speed you need.
+        </p>
+      </section>
+
+      <section class="compare-bikes" id="compare-bikes">
+        <h2 class="section-title">Compare bikes</h2>
+        <div class="products">
+          <article class="products__item item">
+            <img
+              class="item__photo image"
+              src="images/sporty_4.png"
+              alt="Sporty 4"
```

Comment:
It is better to provide a more detailed description for the `alt` attribute, for example `Bike model Sporty 4`